primitives/arithmetic/src/fixed_point.rs

```diff
@@ -372,6 +372,23 @@ macro_rules! implement_fixed {
 			}
 		}
 
+		impl $name {
+			/// const version of `FixedPointNumber::from_inner`.
+			pub const fn from_inner(inner: $inner_type) -> Self {
+				Self(inner)
+			}
+
+			#[cfg(any(feature = "std", test))]
+			pub fn from_fraction(x: f64) -> Self {
+				Self((x * (<Self as FixedPointNumber>::DIV as f64)) as $inner_type)
```

Comment:
I am confused why when I used `$div` this did not work 🤔 
```suggestion
				Self((x * ($div as f64)) as $inner_type)
```